components/uniset/benches/inv_list.rs

```diff
@@ -0,0 +1,59 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use icu_unicodeset::UnicodeSet;
+use std::convert::TryFrom;
+
+/// Best Case Contains
+///
+/// Create a single small range and check contains on every value in range
+fn best_case_contains(c: &mut Criterion) {
+    let check = vec![65, 70];
+    let uset = UnicodeSet::try_from(check).unwrap();
+    c.bench_function("inv_list/contains_best", |b| {
+        b.iter(|| uset.iter().map(|c| uset.contains(c)))
```

Comment:
We're going with  `try_from::<Vec<u32>>` right now since it will come from the data-provider, and ideally people would be serializing their dataset using ICU4C/J but also with the upcoming serializer in L2. 

I agree it is a little confusing as `uset.iter()` actually returns `char` not `u32` (`contains` publicly only works for `char`)

Reply:
agreed and changed regarding the usage of `c`